crates/types/src/config/common.rs

```diff
@@ -385,11 +436,29 @@ impl Default for CommonOptions {
                 Some(15),
                 Some(Duration::from_secs(5)),
             ),
-            auto_provision_partitions: true,
+            cluster_configuration: ClusterConfigurationType::default(),
+            default_cluster_configuration: None,
         }
     }
 }
 
+/// # ClusterConfigurationType
+///
+/// Configure initial cluster configuration.
```

Comment:
This sentence reads very clumsily and only raises more questions. For example, is there a cluster configuration other than the initial one? If the heading tells the full story, we don't need to add more here, e.g. "# Cluster configuration" might be just enough?

Reply:
You are absolutely right.